.github/workflows/plugins_react_tests.yml

```diff
@@ -13,27 +13,43 @@ permissions:
   contents: read
 
 jobs:
+  setup_matrix:
+    name: Setup matrix
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.build_matrix.outputs.matrix }}
+    steps:
+      - name: Build test matrix
+        id: build_matrix
+        run: |
+          curl --silent --show-error --fail --output matrix.json https://raw.githubusercontent.com/theforeman/foreman/${{ github.base_ref }}/.github/matrix.json
+          cat >matrix_include.json <<EOF
+          [{"repo": "foreman-tasks", "org": "theforeman", "ruby": "2.7"}, {"repo": "katello", "org": "Katello"}]
```

Comment:
I think `include` won't do here. Isn't this essentially adding another vector to the matrix?

Perhaps it would also be easier if we simply had `repo` as fully qualified. So `theforeman/foreman-tasks` and `Katello/katello`. For the exact paths it doesn't matter and makes other things easier.